src/lib/themes.js

```diff
@@ -166,11 +166,25 @@ export const normalizeTheme = (data = {}) => {
   }
 
   if (images.headerURL) {
-    const background = normalizeThemeBackground(images.headerURL);
+  // Fx69 update - deprecate headerURL
+    images.theme_frame = images.headerURL;
+    const background = normalizeThemeBackground(images.theme_frame);
     if (background) {
       theme.images.additional_backgrounds = [background];
     }
   }
+  // Fx69 update - deprecate accentcolor
+  if (theme.colors.accentcolor) {
+    theme.colors.frame = theme.colors.accentcolor;
+  }
+ // Fx69 update - deprecate textcolor
```

Comment:
Ditto :)

Reply:
This one looks ok to me?

Reply:
Oh, I see... do you mean indentation again? Got it.